### PR TITLE
Jira graph handlers

### PR DESCRIPTION
**Pull request by damienomurchu**
PR related to several JIRAs & the implementation of key graph dataset handler methods:

- https://issues.jboss.org/browse/FH-3469
- https://issues.jboss.org/browse/FH-3470
- https://issues.jboss.org/browse/FH-3471
- https://issues.jboss.org/browse/FH-3472
- https://issues.jboss.org/browse/FH-3473

### CODE REVIEW

File: utils/averageage.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    var numIssues = relevantIssues.length; // get number of issues
+
+    // get age of each issue & derive average age of issues
+    dataset[date] = relevantIssues.map(iss => {
+      var created = moment(iss.created).add(12, 'hours'); // hack as date 1 hour off
+      var current = moment(date).add(12, 'hours'); // hack as date 1 hour off
+      return current.diff(created, 'days');
+    }).reduce((a, b) => {
+      return a + b;
+    }) / numIssues;
```
Comment (austincunningham) [inline utils/averageage.js:37]: Don't recognize this syntax may be a type O of comment

**Review by austincunningham — COMMENTED** [review]
must pull and run

File: utils/averageage.js (hunk 1 above)
Comment (damienomurchu) [reply utils/averageage.js:37]: which part of the syntax? ` / numIssues` is dividing the total age of all tickets in the previous line by the total number of tickets

**Review by austincunningham — APPROVED** [review]
Yep it runs

**Pull request merged**